### PR TITLE
Projects panel

### PR DESCRIPTION
**Pull request by MrMarchello**

### CODE REVIEW

File: src/app/main/project-panel/project.service.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    function init() {
+      var isInitialized = false;
+
+      firebaseReferenceService
+        .getUserProjectsRef()
+        .on("value", function(projectList) {
+          _projectList = projectList.val();
+
+          if (!isInitialized) {
+            _projectList.myFarm.isActive = true;
```
Comment (sebastianovide) [inline src/app/main/project-panel/project.service.js]: some error ?

File: src/app/main/project-panel/project-panel.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<section class="my-projects">
+  <md-menu>
+    <md-button class="md-warn md-raised md-hue-2" ng-click="vmProjectPanel.openMenu($mdOpenMenu, $event)">
+      My projects
+    </md-button>
+
+    <md-menu-content width="4">
+      <md-menu-item ng-repeat="project in vmProjectPanel.getProjectList()">
+        <md-button ng-click="vmProjectPanel.toggleProject(project)">
+          <i class="fa fa-toggle-{{ project.isActive ? 'on' : 'off' }}"></i>
```
Comment (sebastianovide) [inline src/app/main/project-panel/project-panel.html]: md-switch

File: src/app/main/project-panel/project-panel.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+          {{ project.projectName }}
+        </md-button>
+      </md-menu-item>
+
+      <md-menu-divider></md-menu-divider>
+
+      <md-menu-item>
+        <md-button ng-click="vmProjectPanel.displayNewProjectModal()">
+          <i class="fa fa-plus-square"></i>
```
Comment (sebastianovide) [inline src/app/main/project-panel/project-panel.html]: md-icon

File: src/app/main/project-panel/project-panel.directive.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      var vm = this;
+
+      projectService.init();
+
+      vm.getProjectList = projectService.getProjectList;
+
+      /**
+       * Deactivates all projects.
+       */
+      vm.deactiveAllProjects = function() {
```
Comment (GeovationMax) [inline src/app/main/project-panel/project-panel.directive.js]: Define externally available methods centrally at the top, define functions only with local variables

**Pull request merged**